src/everest/optimizer/opt_model_transforms.py

```diff
@@ -109,35 +126,68 @@ def linear_constraints_to_optimizer(
     ) -> tuple[NDArray[np.float64], NDArray[np.float64], NDArray[np.float64]]:
         r"""Transform a set of linear constraints.
 
-        The set of linear constraints can be represented by a matrix equation:
-        $\mathbf{A} \mathbf{x} = \mathbf{b}$. When linearly transforming
-        variables to the optimizer domain, the coefficients ($\mathbf{A}$) and
-        right-hand-side values ($\mathbf{b}$) must be converted to remain valid.
+        The transformation consists of two steps:
+
+        1. Transformation to correct for variable scaling:
+
+            The set of linear constraints can be represented by a matrix
+            equation: $\mathbf{A} \mathbf{x} = \mathbf{b}$. When linearly
+            transforming variables to the optimizer domain, the coefficients
+            ($\mathbf{A}$) and right-hand-side values ($\mathbf{b}$) must be
+            converted to remain valid.
+
+            Here, the linear transformation of the variables to the optimizer
+            domain is given by the scaling factors $\mathbf{s}_i$ and the
+            offsets $\mathbf{o}_i$:
+
+            $$
+            \hat{\mathbf{x}}_i = \frac{\mathbf{x}_i - \mathbf{o}_i}{\mathbf{s}_i}
```

Comment:
I don't think this is rendered anywhere, as our sphinx-build setup does not seem to include the API (?) (fixing that is of course a separate issue)

Reply:
(but if we intend not to render it anywhere, LaTeX markup is perhaps not appropriate, though beautiful)